infrastructure/build/fab/templaterator.py

```diff
@@ -35,31 +35,24 @@ def check_available(self) -> bool:
 
         return True
 
-    def run(self, input_template: Path,
-            output_file: Path,
-            key_values: Dict) -> None:
-        # pylint: disable=arguments-differ
-        '''
-        This wrapper creates the proper command line options for
-        the Templaterator.
-
-        :param Path input_template: the path to the input template.
-        :param Path output_file: the output file path.
-        :param Dict key_values: the keys and values for the keys to
+    def process(self, input_template: Path,
```

Comment:
•	Disabling the style checker rule doesn't make the problem go away. As noted previously, I appreciate that we are working with a broken abstraction but the Rsync solution is probably the best of a bad lot. To prevent people causing hell by calling {{{run}}} I would override it (with the correct argument list) and throw an exception. {{{NotImplementedError}}} might be appropriate.

Reply:
I have renamed the method. I am not sure what you mean with broken abstraction here?